pkg/sql/catalog/ingesting/write_descs.go

```diff
@@ -51,7 +52,9 @@ func WriteDescriptors(
 	extra []roachpb.KeyValue,
 	inheritParentName string,
 	includePublicSchemaCreatePriv bool,
+	deprecatedAllowCrossDatabaseRefs bool,
```

Comment:
nit: would this variable be more clear if it were named `allowCrossDatabaseRefs`?

can we mention in the comment that the flag is only present since we need to continue supporting RESTOREs of clusters that had cross DB references before they were disallowed?